File: activities/calculadora/calculadora.hs
soma :: Int -> Int -> Int
soma a b = a + b

subtracao :: Int -> Int -> Int
subtracao a b = if a >= b then a - b else b - a

multiplicacao :: Int -> Int -> Int
multiplicacao a b = a * b

divisao :: Float -> Float -> Float
divisao a b = if a >= b then a / b else b / a

main :: IO ()
main = do
    putStrLn "Digite o primeiro número:"
    input1 <- getLine
    let a = read input1 :: Int

    putStrLn "Digite o segundo número:"
    input2 <- getLine
    let b = read input2 :: Int

    putStrLn $ "Soma: " ++ show (soma a b)
    putStrLn $ "Subtração: " ++ show (subtracao a b)
    putStrLn $ "Multiplicação: " ++ show (multiplicacao a b)
    putStrLn $ "Divisão: " ++ show (divisao (fromIntegral a) (fromIntegral b))

File: activities/Calculadora/calculadora.c
#include <stdio.h>
#include <stdlib.h>

int soma(int a, int b) {
    return a + b;
}

int subtracao(int a, int b) {
    if (a >= b)
        return a - b;
    else
        return b - a;
}

int multiplicacao(int a, int b) {
    return a * b;
}

float divisao(float a, float b) {
    if (a >= b)
        return a / b;
    else
        return b / a;
}

int main() {

    int a,b;

    printf("Didite o primeiro número: ");
    scanf ("%d", &a);
    
    printf("Didite o segundo número: ");
    scanf ("%d", &b);

    printf("Soma: %d \n",soma(a,b));
    printf("Subtração: %d \n",subtracao(a,b));
    printf("Multiplicação: %d \n",multiplicacao(a,b));
    printf("Divisão : %1.1f \n",divisao(a,b));

}